images/macos/helpers/Xcode.Helpers.psm1

```diff
@@ -121,4 +121,29 @@ function Get-XcodePairsList {
         $result += "$watchName $phoneName"
     }
     return $result
+}
+
+function Invoke-XCVersion {
+    param(
+        [Parameter(Mandatory)]
+        [string] $Command,
```

Comment:
May be we can pass only arguments to function and concat them with `"xcversion $Arguments"`

Reply:
Done